### DISCUSSION TITLE
cant install `@siteimprove/alfa-react`

### DISCUSSION THREAD

**Discussion by noahehall** [Siteimprove/alfa#832] [Questions] [answered]
`npm i @siteimprove/alfa-react`

returns `npm ERR! code E404 not found`

**Comment by kasperisager** [comment, marked as answer]
Alfa is distributed through GitHub packages and so to install the packages you'll have to configure npm to pull `@siteimprove` packages from GitHub: https://github.com/Siteimprove/alfa#usage

**Reply by kasperisager** [reply]
See #608 for a similar discussion.

**Reply by kasperisager** [reply]
I'll go ahead and accept this answer. Let me know in case it doesn't solve your issue!
